sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -411,6 +411,42 @@ object RapidsConf {
       .integerConf
       .createWithDefault(2)
 
```

Comment:
we may want to add some dev documentation if we are recommending this be done differently on YARN where log aggregation could handle moving core dump.